scripts/links_checker/test/arguments/parser/links_checker_arguments_parser_test.dart

```diff
@@ -22,23 +29,68 @@ void main() {
     );
 
     test(
-      ".parseArgResults() parses the given paths option value to a links checker arguments instance with paths",
+      ".configureArguments() configures the given arg parser to accept the ignore option",
       () {
-        const paths = 'file1 path/to/file2';
-        const options = ['--paths', paths];
+        final parser = ArgParser();
+
+        argumentsParser.configureArguments(parser);
 
-        final expected = ['file1', 'path/to/file2'];
+        final options = parser.options;
 
+        expect(options, contains(LinksCheckerArgumentsParser.ignorePaths));
+      },
+    );
+
+    test(
+      ".configureArguments() configures the given arg parser to have an empty string as a default paths argument value",
+      () {
         final parser = ArgParser();
-        const argumentsParser = LinksCheckerArgumentsParser();
 
         argumentsParser.configureArguments(parser);
 
-        final argResults = parser.parse(options);
-        final arguments = argumentsParser.parseArgResults(argResults);
+        final options = parser.options;
+        final pathsOption = options[LinksCheckerArgumentsParser.paths];
 
-        expect(arguments.paths, equals(expected));
+        expect(pathsOption.defaultsTo, equals(''));
+      },
+    );
+
+    test(
+      ".parseArgResults() creates a links checker arguments instance with paths equal to an empty list if the given paths argument equals an empty string",
+      () {
+        final arguments = argumentsParser.parseArgResults(argResultsMock);
+
+        expect(arguments.paths, equals([]));
+      },
+    );
+
+    test(
+      ".parseArgResults() creates a links checker arguments instance with ignorePaths equal to an empty list if the given ignore paths argument equals an empty string",
+      () {
+        final arguments = argumentsParser.parseArgResults(argResultsMock);
+
+        expect(arguments.ignorePaths, equals([]));
+      },
+    );
+
+    test(
+      ".parseArgResults() creates a links checker arguments instance with paths equals to an empty list",
+      () {
+        final arguments = argumentsParser.parseArgResults(argResultsMock);
+
+        expect(arguments.paths, equals([]));
+      },
+    );
+
+    test(
+      ".parseArgResults() creates a links checker arguments instance with ignore paths equals to an empty list",
+      () {
+        final arguments = argumentsParser.parseArgResults(argResultsMock);
+
+        expect(arguments.ignorePaths, equals([]));
```

Comment:
Should we test here that the argument parser correctly parses the given `paths` and `ignorePaths` arguments?